src/config/capabilityStatementResources.json

```diff
@@ -0,0 +1,267 @@
+{
+  "mode": "server",
+  "resource": [
+    {
+      "type" : "Library",
+      "profile" : "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/library-cqfm",
+      "supportedProfile" : [
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/computable-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/publishable-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/executable-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/modelinfo-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/quality-program-cqfm"
+      ],
+      "interaction" : [
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "code" : "read",
+          "documentation" : "Read allows clients to get the definitions and details of repository libraries"
+        },
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "code" : "search-type",
+          "documentation" : "Search allows clients to search for repository libraries"
+        }
+      ],
+      "searchParam" : [
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "name" : "url",
+          "definition" : "http://hl7.org/fhir/SearchParameter/Library-url",
+          "type" : "uri"
+        },
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "name" : "version",
+          "definition" : "http://hl7.org/fhir/SearchParameter/Library-version",
+          "type" : "token"
```

Comment:
I believe that "version" is of type string. From the docs, it looks like token search params are typically used for parameters of code or identifier type. Since version is just of type string, I think this should just be string also?

Reply:
It looks like the type is string in the Capability Statement docs but then in the reference implementation its of type token ❓

Reply:
Oops yes I agree with you Nat that this should be of type "string" so I'll make that change. The Shareable MR capability statement lists it as a token which seems to be incorrect. Thanks for doing that cross-checking!

Reply:
Ah I actually found something interesting completely by accident looking through the spec for my other task so I could be wrong. But in the FHIR search docs, it says "A token type is a parameter that provides a close to exact match search on a string of characters, potentially scoped by a URI" which I believe is what we want here since search for version is an exact match but search for other string types (title, name, description) are not exact matches but use FHIR string search semantics (https://build.fhir.org/search.html#token)

In addition, there's this:
![image](https://user-images.githubusercontent.com/30158156/215820509-7c0e08b1-9e04-4a0b-b963-f05546c2e04d.png)

I could be wrong about this, but wanted to bring it up for discussion.